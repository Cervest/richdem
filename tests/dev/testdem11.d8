ncols         15
nrows         15
xllcorner     421568
yllcorner     4872699
cellsize      3
NODATA_value  9
0 0 0 0 5 7 0 0 0 0 0 0 0 0 0
0 0 0 0 7 1 0 0 0 0 0 0 0 0 0
0 0 0 0 5 7 0 0 7 1 1 1 1 1 1
0 0 0 0 7 1 0 0 7 0 0 0 0 0 0
0 0 0 0 5 7 0 0 7 0 0 0 0 0 0
0 0 0 0 7 1 0 0 7 0 0 0 0 0 0
0 0 0 0 5 7 0 0 7 0 0 0 0 0 0
0 0 0 0 7 1 0 0 5 5 5 7 0 0 0
0 0 0 0 5 7 0 7 1 0 0 7 0 0 0
0 0 0 0 7 1 0 7 3 0 0 7 0 0 0
0 0 0 0 5 7 0 7 3 0 0 7 0 0 0
0 0 0 0 7 1 0 7 3 0 0 7 0 0 0
0 0 0 0 5 7 0 7 3 1 1 1 0 0 0
0 0 0 0 7 1 0 7 0 0 0 0 0 0 0
0 0 0 0 5 7 0 7 0 0 0 0 0 0 0