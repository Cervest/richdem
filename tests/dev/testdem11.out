   1   1   1   1   1   2   1   1   1   1   1   1   1   1   1
   1   1   1   1   4   3   1   1   1   1   1   1   1   1   1
   1   1   1   1   5   6   1   1   7   6   5   4   3   2   1
   1   1   1   1   8   7   1   1   8   1   1   1   1   1   1
   1   1   1   1   9  10   1   1   9   1   1   1   1   1   1
   1   1   1   1  12  11   1   1  10   1   1   1   1   1   1
   1   1   1   1  13  14   1   1  11   1   1   1   1   1   1
   1   1   1   1  16  15   1   1  12  13  14  15   1   1   1
   1   1   1   1  17  18   1  28  27   1   1  16   1   1   1
   1   1   1   1  20  19   1  29  26   1   1  17   1   1   1
   1   1   1   1  21  22   1  30  25   1   1  18   1   1   1
   1   1   1   1  24  23   1  31  24   1   1  19   1   1   1
   1   1   1   1  25  26   1  32  23  22  21  20   1   1   1
   1   1   1   1  28  27   1  33   1   1   1   1   1   1   1
   1   1   1   1  29  30   1  34   1   1   1   1   1   1   1
